server/cmwell-cons/extract-scala.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+cd $(dirname -- "$0")
+bash -c "
+cd /home/u/cmwell/components-extras;
+tar -xf scala-2.12.6.tgz;
+mv scala-2.12.6 scala;
+
+" 2> /dev/null
```

Comment:
why do u added this file?what is the purpose?

Reply:
Redundant. Removed....

Reply:
Sorry - readied. This is required to extract the scala tar.gz file

Reply:
why didn't u use the exisitng extract-java-scala that we already have in cmwell-cons?